ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -156,6 +165,17 @@ SettingsContentBase {
             id: stackLayout
             currentIndex: profileTabBar.currentIndex
 
+            onCurrentIndexChanged: {
+                if(root.profileStore.isFirstShowcaseInteraction &&
+                   (currentIndex === MyProfileView.TabIndex.Communities ||
+                   currentIndex === MyProfileView.TabIndex.Accounts ||
+                   currentIndex === MyProfileView.TabIndex.Collectibles ||
+                   currentIndex === MyProfileView.TabIndex.Assets)) {
```

Comment:
What about the `Web` tab? :)

Reply:
I'd do it the opposite way, `if not (identity tab)`

Reply:
The web tab is excluded in purpose bc it's what I understood from the requirements.. but let me ask `Ben`.

Reply:
Checked! You are right. I'll update to include the web tab! Thanks for pointing it out!